packages/postman-to-openapi/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@scalar/postman-to-openapi",
+  "description": "Converts Postman collections to OpenAPI documents",
+  "license": "MIT",
+  "author": "Scalar (https://github.com/scalar)",
+  "homepage": "https://github.com/scalar/scalar",
+  "bugs": "https://github.com/scalar/scalar/issues/new/choose",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/scalar/scalar.git",
+    "directory": "packages/postman-to-openapi"
+  },
+  "keywords": [
+    "postman",
+    "openapi",
+    "swagger",
+    "converter",
+    "transformer",
+    "export",
+    "scalar"
+  ],
+  "version": "0.0.0",
+  "engines": {
+    "node": ">=18"
+  },
+  "scripts": {
+    "build": "scalar-build-rollup",
+    "format": "scalar-format-js",
+    "lint:check": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "test": "vitest",
+    "types:build": "scalar-types-build",
+    "types:check": "scalar-types-check",
+    "watch": "scalar-build-rollup -w"
+  },
+  "type": "module",
+  "main": "dist/index.js",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
+    },
+    "./css/*.css": {
+      "import": "./dist/css/*.css",
+      "require": "./dist/css/*.css",
+      "default": "./dist/css/*.css"
+    },
+    "./*.css": {
+      "import": "./dist/*.css",
+      "require": "./dist/*.css",
+      "default": "./dist/*.css"
+    }
```

Comment:
Should be no reason to export CSS. Might need to add a setting on the rollup config

Reply:
I thought there was an includeCss flag or something